libavfilter/vf_scale_vaapi.c

```diff
@@ -85,15 +87,19 @@ static int scale_vaapi_config_output(AVFilterLink *outlink)
     ff_scale_adjust_dimensions(inlink, &vpp_ctx->output_width, &vpp_ctx->output_height,
                                ctx->force_original_aspect_ratio, ctx->force_divisible_by);
 
-    if (inlink->w == vpp_ctx->output_width && inlink->h == vpp_ctx->output_height &&
+    if (ctx->passthrough && 
```

Comment:
let's call it enable_passthrough to differentiate it from the other passthroughs

Reply:
Done.